v03_pipeline/lib/reference_datasets/exac.py

```diff
@@ -1,10 +1,11 @@
 import hail as hl
 
+from v03_pipeline.lib.misc.nested_field import parse_nested_field
 from v03_pipeline.lib.model import ReferenceGenome
 from v03_pipeline.lib.reference_datasets.misc import vcf_to_ht
 
-RENAME = {
-    'AF_POPMAX': 'info.AF_POPMAX',
+SELECT = {
+    'AF_POPMAX': 'info.POPMAX',
```

Comment:
This field change comes from us using the `vcf` rather than the `ht`.  Test caught it!